From 9f98fed28d6be03a4d352afdf60883d286b5f820 Mon Sep 17 00:00:00 2001
From: Tim Gover <tim.gover@raspberrypi.com>
Date: Fri, 1 Nov 2024 19:42:17 +0000
Subject: [PATCH 681/697] raspberrypi-firmware: Add the RPI firmware UART APIs

Add VideoCore mailbox definitions for the new RPi firmware UART.

Signed-off-by: Tim Gover <tim.gover@raspberrypi.com>
---
 include/soc/bcm2835/raspberrypi-firmware.h | 2 ++
 1 file changed, 2 insertions(+)

--- a/include/soc/bcm2835/raspberrypi-firmware.h
+++ b/include/soc/bcm2835/raspberrypi-firmware.h
@@ -98,6 +98,8 @@ enum rpi_firmware_property_tag {
 	RPI_FIRMWARE_GET_REBOOT_FLAGS =                       0x00030064,
 	RPI_FIRMWARE_SET_REBOOT_FLAGS =                       0x00038064,
 	RPI_FIRMWARE_NOTIFY_DISPLAY_DONE =                    0x00030066,
+	RPI_FIRMWARE_GET_SW_UART =                            0x0003008a,
+	RPI_FIRMWARE_SET_SW_UART =                            0x0003808a,
 
 	/* Dispmanx TAGS */
 	RPI_FIRMWARE_FRAMEBUFFER_ALLOCATE =                   0x00040001,
